### DISCUSSION TITLE
[@types/react-router-dom] - Routes missing and Switch still exists??

### DISCUSSION THREAD

**Discussion by one-dumb-dude** [DefinitelyTyped/DefinitelyTyped#57819] [Issues with a @types package]
am I missing something here.. kinda new to typings.. but it seems version @types/react-router-dom version 5.3.2 still has the 'Switch' class from 'react-router'... Currently the switch library is no longer in use.. instead its been replaced with Routes class. will there be an update? or how do i update my something on my end here?

[a comment by an automated account is not shown]

**Comment by 1pete** [comment]
Sorry, I don't quite understand. Can you share the code where you have your problem?
